spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/RegionProposal.scala

```diff
@@ -143,9 +143,7 @@ class RegionProposal(
       val postNmsTopN = if (this.isTraining()) min(postNmsTopNTrain, bboxNumber)
       else min(postNmsTopNTest, bboxNumber)
 
-      if (output.getOrElse(b, null) == null) {
```

Comment:
why delete this? output the output this a table, we can reuse the tensors in the output.

Reply:
Here is a warning of "comparing values of types Null and Null using `==' will always yield true "

Reply:
Use output.contains(b) instead